<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectionWindow</class>
 <widget class="QMainWindow" name="SelectionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>633</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1019</width>
    <height>633</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1019</width>
    <height>633</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Thermodynamic Simulator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(79, 29, 11)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1011</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code SemiBold</family>
      <pointsize>28</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255,255,255);</string>
    </property>
    <property name="text">
     <string>Select Cycle:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Brayton_Image">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>311</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../graphics/brayton_cycle.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Diesel_Image">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>341</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../graphics/diesel_cycle.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Otto_Image">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>80</y>
      <width>311</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../graphics/otto_cycle.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="Brayton_Button">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>420</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code SemiBold</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255); background-color: rgb(95, 16, 3)</string>
    </property>
    <property name="text">
     <string>Brayton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Diesel_Button">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>420</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code SemiBold</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255); background-color: rgb(95, 16, 3)</string>
    </property>
    <property name="text">
     <string>Diesel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Otto_Button">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>420</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code SemiBold</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255); background-color: rgb(95, 16, 3)</string>
    </property>
    <property name="text">
     <string>Otto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Help_Button">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>550</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Fira Code SemiBold</family>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255); background-color: rgb(95, 16, 3)</string>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
